### PR TITLE
Feature/consensus/stop

### PR DESCRIPTION
**Pull request by minsulee2**
Changes:

- Safe exit process with SIGINT
- Now support socket.on('close')

### CODE REVIEW

File: tracker-server/index.js [hunk 1]
```diff
@@ -124,7 +124,7 @@ server.on('connection', (ws) => {
 
   ws.on('close', (code) => {
     logger.info(`\nDisconnected from node [${node ? abbrAddr(node.address) : 'unknown'}] ` +
-        `with code: ${code}`);
+                `with code: ${code}`);
```
Comment (liayoo) [inline tracker-server/index.js:127]: remove spaces?

File: client/index.js [hunk 2]
```diff
@@ -20,6 +20,14 @@ process.on('uncaughtException', function (err) {
   logger.error(`[${CLIENT_PREFIX}]` + err);
 });
 
+process.on('SIGINT', _ => {
+  logger.info("############################################################");
+  logger.info("##          Stopping the blockchain client....            ##");
+  logger.info("############################################################");
```
Comment (liayoo) [inline client/index.js:26]: (suggestion) can we make this into one log?

File: server/index.js [hunk 3]
```diff
@@ -372,7 +393,7 @@ class P2pServer {
     });
 
     socket.on('close', () => {
```
Comment (liayoo) [inline server/index.js:395]: Could you leave a todo for handling/recording disconnections and errors here?

Comment (minsulee2) [reply server/index.js:396]: I left TODO here!

File: tracker-server/index.js (hunk 1 above)
Comment (minsulee2) [reply tracker-server/index.js]: I got this

File: client/index.js (hunk 2 above)
Comment (minsulee2) [reply client/index.js]: made it smaller :)

**Comment by minsulee2** [comment]
Thanks for the review!

**Pull request merged**